mp/src/game/shared/momentum/weapon/weapon_mom_stickybomblauncher.cpp

```diff
@@ -238,6 +243,10 @@ void CMomentumStickybombLauncher::SecondaryAttack()
     if (m_iStickybombCount == 0)
         return;
 
+    float flTimeFrom2Det = gpGlobals->curtime - m_fl2DetCooldownBeginTime;
+    if (flTimeFrom2Det < mom_sj_det_buffer_ticks.GetInt() * 0.015f + MOM_STICKYBOMB_TICK_EPSILON)
```

Comment:
Use `gpGlobals->interval_per_tick` instead of the `0.015f` constant

Reply:
Will do. I should note that if  testers want this and it is merged into dev, I think the value of 'mom_sj_det_buffer_ticks.GetInt() * 0.015f + MOM_STICKYBOMB_TICK_EPSILON' should be replaced with the hard coded time calculated the tick count chosen (e.g if 5 ticks is chosen, this should be 0.075 + epsilon, not  5 * gpGlobals->interval_per_tick + epsilon). This shouldn't be a modifiable duration  if it becomes part of the game (at least without sv_cheats) and the actual duration shouldn't change due to tick rate - I only have it based on ticks so that people don't waste time testing float value changes less than a TF2 tick which would result in no change.

Reply:
Yep agreed, but for this experiment it will be changeable.